sdk/identity/Azure.Identity/src/AzureIdentityEventSource.cs

```diff
@@ -50,6 +52,7 @@ internal sealed class AzureIdentityEventSource : AzureEventSource, IIdentityLogg
         internal const string UserAssignedManagedIdentityNotSupportedMessage = "User assigned managed identities are not supported in the {0} environment.";
         internal const string ServiceFabricManagedIdentityRuntimeConfigurationNotSupportedMessage = "Service Fabric user assigned managed identity ClientId or ResourceId is not configurable at runtime.";
         internal const string ManagedIdentitySourceAttemptedMessage = "ManagedIdentitySource {0} was attempted. IsSelected={1}.";
+        internal const string ManagedIdentityCredentialSelectedMessage = "Managed Identity source selected: {0} with ID: {1}";
```

Comment:
Nits:
- Should this constant be named `ManagedIdentitySourceSelectedMessage`?
- Should the "Managed identity source" string become "ManagedIdentitySource" to match what's used in the constant string on line 54?

Reply:
It's internal, so the naming doesn't really have much impact.